android/app/src/main/res/layout/activity_main.xml

```diff
@@ -4,29 +4,14 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <com.google.android.material.appbar.AppBarLayout
-        android:id="@+id/appBar"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:fitsSystemWindows="true"
-        app:layout_constraintTop_toTopOf="parent">
-
-        <androidx.appcompat.widget.Toolbar
-            android:layout_width="match_parent"
-            android:layout_height="?actionBarSize"
-            android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"
-            app:title="@string/app_name" />
-
-    </com.google.android.material.appbar.AppBarLayout>
```

Comment:
Will we display the Toolbar on the Home screen? 🤔  If so, we will add ToolBar from Fragment's scope instead of Activity? 🤔

Reply:
Yes. Or you would like to have the toolbar on the Login screen also? I can update.

Reply:
I mean we should keep the toolbar as a placeholder on the activity. Then we could update the toolbar when the user navigates to the screen.

Reply:
Hmm, I'm not sure I understand you correctly, Is that you'd like to keep it and hide it first, then if have need it we will show it programmatically? @hoangnguyen92dn